app/javascript/components/editor/TestsGroupList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { TestStatus, Test } from './types'
+import { GraphicalIcon } from '../common/GraphicalIcon'
+import { TestsGroup } from './TestsGroup'
+
+export function TestsGroupList({ tests }: { tests: Test[] }): JSX.Element {
+  const testsWithIndex = tests.map((test, i) => ({ index: i + 1, ...test }))
+  const firstFailedTestIdx = testsWithIndex.findIndex(
+    (test) =>
+      test.status === TestStatus.FAIL || test.status === TestStatus.ERROR
+  )
+  const passed = testsWithIndex.filter(
+    (test) => test.status === TestStatus.PASS
+  )
+  const failed =
```

Comment:
You could consider building `passed` and `failed` the same way, and then using `failed[0]` to get the first failed test, instead.

Reply:
Thanks. I did this :)